server/controllers/validators/messageControllerValidator.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @Author:     Alex Qin
+ * @Created:    2021.03.23
+ *
+ * @Description: functions to validate input to controller functions to send a message
+ *
+ */
+
+const { body } = require('express-validator/check');
+const {
+    isValidMessage,
+    uidShouldExistAndBeForAMember
+} = require('./messageControllerValidatorUtils');
+
+const messageContentValidation = [
+    body('content', "A message must be provided")
+        .exists()
+        .custom(content => isValidMessage(content)),
```

Comment:
When validating input, you also have to sanitize it - this will protect us against things like SQL injection attacks. For strings you can use the following validation & sanitation:
![image](https://user-images.githubusercontent.com/43148498/112730727-630ca380-8ef0-11eb-8aa1-5d417cc9caaf.png)

Reply:
You can read `express-validator` documentation for more information about what each of these functions do

Reply:
Addressed.